Added possibility to choose VM in multi vApp template (#501)
terraform-providers/terraform-provider-vcd: website/docs/r/vapp_vm.html.markdown

```

## Example Usage (vApp template with multi VMs)

> This example shows how to create a VM using vApp template with multi VMs.

Reviewer: New line seems to be missing between header and this sentence. Also, suggest clarifying more:

```suggestion

This example shows how to create a VM from a vApp template with multiple VMs by specifying which VM to use.
```

Author: updated, also there isn't need for new line as Paragraph notions are there and also this inline with other examples.

```hcl
resource "vcd_vapp_vm" "secondVM" {

>   vapp_name           = vcd_vapp.vcd_vapp.web.name

Author: fixed. ty

  name                = "secondVM"
  computer_name       = "db-vm"
  catalog_name        = "cat-where-is-template"
  template_name       = "vappWithMultiVm"

>   vm_name_in_template = "secondVM"

Reviewer: ```suggestion
  vm_name_in_template = "secondVM" # Specifies which VM to use from the template
```

Author: updated

* `computer_name` - (Optional; *v2.5+*) Computer name to assign to this virtual machine. 
* `catalog_name` - (Optional; *v2.9+*) The catalog name in which to find the given vApp Template or media for `boot_image`.
* `template_name` - (Optional; *v2.9+*) The name of the vApp Template to use

> * `vm_name_in_template` - (Optional; *v2.9+*) The name of the VM in vApp Template to use. In cases when vApp template has more than one VM.

Reviewer: ```suggestion
* `vm_name_in_template` - (Optional; *v2.9+*) The name of the VM in vApp Template to use. For cases when vApp template has more than one VM.
```

Author: updated
